autonomi/src/client/data.rs

```diff
@@ -218,26 +220,33 @@ impl Client {
         info!("Getting chunk: {addr:?}");
 
         let key = NetworkAddress::from_chunk_address(ChunkAddress::new(addr)).to_record_key();
-
+        debug!("Converted chunk address to network record key: {:?}", key);
         let get_cfg = GetRecordCfg {
             get_quorum: Quorum::One,
             retry_strategy: None,
             target_record: None,
             expected_holders: HashSet::new(),
             is_register: false,
         };
+        debug!("GetRecordCfg created: {:?}", get_cfg);
 
         let record = self
             .network
             .get_record_from_network(key, &get_cfg)
             .await
             .inspect_err(|err| error!("Error fetching chunk: {err:?}"))?;
+        debug!("Record fetched from network: {:?}", record);
```

Comment:
We should not log `record` as it will be pure Bytes and unreadable. You should probably log the `key` or `address` here